update-cache/README.md

```diff
@@ -0,0 +1,25 @@
+#AMP-Toolbox update-cache
+
+Generates update-cache invalidation URLs, as described on the AMP documentation at 
+https://developers.google.com/amp/cache/update-ping#update-cache-request.
+
+## Usage
+```javascript
+  const createUpdateCacheUrlFactory = require('amp-toolbox-update-cache');
+
+  // Load the private key, generated as described on 
+  // https://developers.google.com/amp/cache/update-cache#rsa-keys
+  const privateKey = '...';
+
+  // Create an instance of the factory using the private key.
+  const updateCacheUrlFactory = createUpdateCacheUrlFactory(privateKey);
+  const timestamp = (Date.now() / 1000) | 0; // timestamp as a UNIX Epoch in seconds.
+
+  const cacheUpdateUrls = await updateCacheUrlFactory.fromOriginUrl('https://www.example.com');
```

Comment:
maybe use a promise instead of await (or wrap it in an async function) - otherwise people will get confused who are not familiar with async/await

Reply:
done.